internal/appconfig/machines.go

```diff
@@ -62,6 +62,72 @@ func (c *Config) ToReleaseMachineConfig() (*fly.MachineConfig, error) {
 	return mConfig, nil
 }
 
+func (c *Config) ToTestMachineConfig(svc *ServiceMachineCheck, origMachine *fly.Machine) (*fly.MachineConfig, error) {
+	var machineEntrypoint []string
+	if len(svc.Entrypoint) > 0 {
```

Comment:
there is subtle difference bettwen `len(svc.Entrypoint) > 0` and `svc.Entrypoint != nil`, the former includes the later, but by checking for the later is more accurate way to check if an entrypoint wasn't set vs set to an empty entrypoint.

Reply:
true! thank you for pointing this out